plugin/oldfilesearch.vim

```diff
@@ -75,6 +75,20 @@ function! s:GetOldFiles(patterns)
         call add(candidates, ffull)
         let oldindex[ffull] = oidx
     endfor
+    " Prepend all regular buffers that are not yet candidates.
+    for l:b in reverse(range(1, bufnr('$')))
+        " skip non-existing and special buffers.
+        if !bufexists(l:b) || !empty(getbufvar(l:b, '&buftype'))
+            continue
+        endif
+        " skip buffers that are already candidates.
+        let bfull = fnamemodify(bufname(l:b), ':p')
```

Comment:
Good catch, thank you!  The directory happens to get excluded later, because it is not a file.  It is still better to fix this up - ce3645e95a2fbb97d60acb47601c901449248172.